session/handler.go

```diff
@@ -124,6 +126,13 @@ type toSession struct {
 	//
 	// in: header
 	Cookie string `json:"Cookie"`
+
+	// Return the session additional in token format like a JWT
```

Comment:
```suggestion
	// Returns the session additionally as a token (such as a JWT)
```

Also, add a link to the docs? (Once they are available)
And a note about the caching behavior?